Sources/Jetworking/Client/Interceptor/Protocol/Interceptor.swift

```diff
@@ -1,4 +1,43 @@
 import Foundation
 
-/// Base Protocol for `RequestInterceptor` and `ResponseInterceptor`.
-public protocol Interceptor {}
+/// Base Protocol for `Interceptor`.
+/// `Interceptors` are able to intercept a request or a response.
```

Comment:
Aren't they allowed to do both? So the comment would suit better like
` `Interceptors` are able to intercept a request and a response.`